releases/v20.1.0-rc.1.md

```diff
@@ -0,0 +1,276 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-rc.1.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.0-rc.1.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.1.0-rc.1
+~~~
+
+### Security updates
+
+- The new `--unencrypted-localhost-http` flag for [`cockroach start`](../v20.1/cockroach-start.html) and [`cockroach start-single-node`](../v20.1/cockroach-start-single-node.html) forces the HTTP listener to bind to `localhost` addresses only and disables the TLS protocol. In secure clusters, this makes the Admin UI reachable with an `http://` URL without requiring certificate or CA setup. [#46472][#46472] {% comment %}doc{% endcomment %}
+
+### General changes
+
+- Transactions reading a lot of data behave better when exceeding the memory limit set by the `kv.transaction.max_refresh_spans_bytes` [cluster setting](../v20.1/cluster-settings.html). Such transactions now attempt to resolve the conflicts they run into instead of being forced to always retry. Increasing `kv.transaction.max_refresh_spans_bytes` should no longer be necessary for most workloads. [#46803][#46803]
+- Schema changes started in v19.2 before [upgrading to v20.1](../v20.1/upgrade-cockroach-version.html) are now now automatically migrated in the background after the upgrade is finalized so that they can run to completion. [#47073][#47073] {% comment %}doc{% endcomment %}
+
+### Enterprise edition changes
+
+- The new `protect_data_from_gc_on_pause` [`CHANGEFEED` option](../v20.1/create-changefeed.html#options) ensures that the data needed to resume a `CHANGEFEED` is not garbage collected. [#46345][#46345] {% comment %}doc{% endcomment %}
+- [`BACKUP`](../v20.1/backup.html) and [`RESTORE`](../v20.1/restore.html) now collect some anonymous telemetry on throughput and feature usage. [#46755][#46755]
+- [`SHOW BACKUP`](../v20.1/show-backup.html) now shows whether a `BACKUP` is a full cluster backup or not. [#46768][#46768] {% comment %}doc{% endcomment %}
+- [`SHOW BACKUP`](../v20.1/show-backup.html)  now shows the privileges assigned to tables and databases in a backup and, if the `WITH privileges` option is specified, lists which users and roles had which privileges. [#46853][#46853] {% comment %}doc{% endcomment %}
+- [`BACKUP`](../v20.1/backup.html) and [`RESTORE`](../v20.1/restore.html) to HTTP storage locations now requires explicitly specifying incremental storage locations. This means HTTP storage is not supported when using the new automatically appended incremental syntax. [#46967][#46967] {% comment %}doc{% endcomment %}
```

Comment:
~Is this a security thing?  Does this only apply to HTTP but not HTTPS?  Asking because I sometimes assume when reading that HTTP refers to the overall protocol, including HTTPS.